packages/headache/headache.1.04/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+name: "headache"
+version: "1.04"
+
+license: "GNU Library General Public License"
+
+synopsis: "Automatic generation of files headers"
+description: """
+Lightweight tool for managing headers in source code files. It can
+update in any source code files (OCaml, C, XML et al).
+"""
+
+authors: [
+  "Vincent Simonet"
+  # contributors
+  "Patrick Baudin"
+  "Mehdi Dogguy"
+  "François Pottier"
+  "Virgile Prevosto"
+  "Ralf Treinen"
+]
+
+maintainer: "Patrick Baudin"
+homepage: "https://github.com/Frama-C/headache/"
+bug-reports: "https://github.com/Frama-C/headache/issues"
+dev-repo: "https://github.com/Frama-C/headache.git"
+
+depends: [
+  "camomile"
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+]
+
+build: [
+  [make]
+]
+
+install: [
```

Comment:
Does the `install` clause do anything different than installing what's already listed in the `headache.install` file that you furnished?

Reply:
We just modify the build state in order to use `dune` that generates such a file. This file is no more needed into the `opam-repository`.

Reply:
ok perfect, then with dune this clause shouldn't be necessary. Could you remove it?

Reply:
The file `headache.install` generated by `dune` is similar to the proposed one. So,
-  `make "INSTALLDIR=%{prefix}%/bin" "install` command installs only the binary listed into `headache.install`
- `make "DOC_INSTALLDIR=%{prefix}%/doc/headache" "install-doc"`  command (executed on the request of `with-doc`) installs the documentation files listed into `headache.install`.

Reply:
Those two commands install things that are already installed through `headache.install` (generated by `dune build`) for what I can see. I'm pretty sure those two commands are unnecessary